design/src/rfcs/rfc0044_disable_payload_signing.md

```diff
@@ -0,0 +1,86 @@
+RFC: Disable Payload Signing
+=============================
+
+> Status: RFC
+>
+> Applies to: client
+
+For a summarized list of proposed changes, see the [Changes Checklist](#changes-checklist) section.
+
+This RFC proposes a way for users to disable payload signing for requests with a streaming binary payload.
+
+
+The user experience if this RFC is implemented
+----------------------------------------------
+
+
+In the current version of the SDK, users are unable to control how a request is signed.
+Most requests must include a hash of the payload as part of the signature. However, requests with a
+[streaming](https://smithy.io/2.0/spec/streaming.html#smithy-api-streaming-trait) `blob` shape as the payload can
+sometimes (e.g. S3 `PutObject`) choose to skip payload hashing and exclude it from the signature. When calculating the
+signature the body hash is instead set to the literal constant `UNSIGNED-PAYLOAD`. This has the advantage of not paying
```

Comment:
What services does this actually work for? I was under the impression this only worked for S3.

Reply:
Yeah, I think we should allow list this until we've verified it actually works in other services.

Reply:
I'm good with that, will update the RFC to mention this.